src/common/config.go

```diff
@@ -248,6 +255,16 @@ func checkConf() error {
 		return fmt.Errorf("Unknown Sandbox type '%s'", Conf.Sandbox)
 	}
 
+	if Conf.Trace.Format != "default" {
+		if Conf.Trace.Format == "text" || Conf.Trace.Format == "json" {
+			if Conf.Trace.Log_file_dir == "" {
+				return fmt.Errorf("must specify log output directory under log_file_dir in config.json")
+			}
+		} else {
+			return fmt.Errorf("unknown log format")
```

Comment:
Best to include the unknown name in the error message

Reply:
tell them exactly what they typed